assets/sass/_cover.scss

```diff
@@ -76,7 +76,7 @@
 			}
 		}
 
-		@media screen and (max-width: $width-tablet) {
+		@include max-width($width-tablet) {
```

Comment:
Expected item on line 79 to appear before line 71. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets